insights/parsers/sysconfig.py

```diff
@@ -630,3 +633,31 @@ class IfCFGStaticRoute(SysconfigOptions):
     def parse_content(self, content):
         self.static_route_name = self.file_name.split("route-", 1)[1]
         super(IfCFGStaticRoute, self).parse_content(content)
+
+
+@parser(Specs.sysconfig_grub)
+class GrubSysconfig(SysconfigOptions):
+    """
+    Class to parse the ``/etc/sysconfig/grub``
```

Comment:
Please add a short explanation that this parser is for `/etc/default/grub`, and the `/etc/sysconfig/grub` is a symlink points to  `/etc/default/grub`.